src/test/resources/features/Non-association.feature

```diff
@@ -55,7 +54,3 @@ Feature: Non-association visit
     Then Im on "Manage prison visits - Select date and time of visit" page
     And I shouldn't be able to select a timeslot that was booked last
     And I sign out of the service
-
-    Examples:
-      | userName  | password          | prisonNumber | prisonerName    | phoneNumber | reason        |
-      | VSIP3_TST | Expired19         | A6038DZ      | VSIP_PRISONER08 | 07805123900 | Health issues |
```

Comment:
Any reasons why we are moving them into the test case? Is this approach not better when it comes to maintaining these tests?

Reply:
We don't need a scenario outline and example table as there is only once scenario here. Example tables should be use only if there is more that one scenario.